cpp/lib/ome/bioformats/FormatReader.h

```diff
@@ -1070,21 +1072,23 @@ namespace ome
        * Get the optimal sub-image width.
        * This is intended for use with openBytes().
        *
+       * @param channel the channel to use.
        * @returns the optimal width.
        **/
       virtual
       dimension_size_type
-      getOptimalTileWidth() const = 0;
+      getOptimalTileWidth(dimension_size_type channel) const = 0;
 
       /**
        * Get the optimal sub-image height.
        * This is intended for use with openBytes().
        *
+       * @param channel the channel to use.
        * @returns the optimal height.
        **/
       virtual
       dimension_size_type
-      getOptimalTileHeight() const = 0;
+      getOptimalTileHeight(dimension_size_type channel) const = 0;
```

Comment:
One counter to using planeIndex rather than channel is that isInterleaved is already using channel rather than planeIndex.